R/textmodel_nb.R

```diff
@@ -216,85 +195,78 @@ textmodel_nb.dfm <- function(x, y, smooth = 1,
 #' predict(nb)
 #' @keywords textmodel internal
 #' @export
-predict.textmodel_nb <- function(object, newdata = NULL, ...) {
+predict.textmodel_nb <- function(object, newdata = NULL, 
+                                 type = c("class", "prob", "link"),
```

Comment:
I called the last one "link" because the name was "log.posterior.link" originally and "glm" also has "link". "logposterior" is two words, so "log.posterior" or "log_posterior" or just "posterior" seems seems better. This is more about the name the list element than the argument.